desktop/appcontainer/AppContainer.ts

```diff
@@ -133,7 +133,7 @@ const appContainerView = hoistCmp.factory({
                     item: frame(createElement(appSpec.componentClass, {model: appModel}))
                 }),
                 inspectorPanel(),
-                versionBar()
+                appModel.supportVersionBar? versionBar() : null
```

Comment:
+ can we incorporate this logic in the internal versionBar() component that is already checking whether it should be shown?

+ Also need to check this in mobile app container as well.